### ISSUE TITLE
webpack issue 'require function is used in a way in which dependencies cannot be statically extracted'

### ISSUE THREAD

**Issue by stheine** [mdevils/html-entities#49]
Hi,

I'm using `html-entities` in a ui application, generated by webpack. when I update from 1.3.1 to 2.0.2, webpack complains with the error message:

```
WARNING in ./node_modules/html-entities/lib/index.js 13:93-94
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./example/src/index.jsx 1:182-216
```

can you help on this issue?

Regards
  stheine

**Comment by mdevils** [comment]
Hello @stheine. Thanks for reporting the problem. I've pushed a fix for this issue, version `2.0.4`.

**Comment by stheine** [comment]
fix confirmed. thanks for the quick action!
